### PR TITLE
Feature/sc 20289/competency client help repo

### PR DESCRIPTION
**Pull request by shanioffe**
Creates the competency-constructor-help repo and initializes it with 6 help pages for the user

### CODE REVIEW

File: editCompetency.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (JosueMolinaMorales) [inline editCompetency.md:1]: These head and tails (line 21) can be deleted.

File: resetPassword.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (JosueMolinaMorales) [inline resetPassword.md:1]: here too

File: submitCompetency.md [hunk 3]
```diff
@@ -1,8 +1,12 @@
+<<<<<<< HEAD
```
Comment (JosueMolinaMorales) [inline submitCompetency.md:1]: and here

**Review by JosueMolinaMorales — CHANGES_REQUESTED** [review]
Just some small changes. Do you know if this is supposed to be a jekyll webpage? Like the cyber4all docs and competency-docs?

**Review by mfranz1 — COMMENTED** [review]
Just Josue's comments

**Review by Cwagne17 — COMMENTED** [review]
Once Sway's comments are addressed this is good

**Review by Maliyaa — APPROVED** [review]

**Review by pmanal1 — COMMENTED** [review]
Looks good. One note: I noticed in the repo that we have duplicate `createCompetency.md` and `searchCompetency.md` files. You should probably delete one of each.

File: editCompetency.md [hunk 4]
```diff
@@ -14,4 +15,7 @@ parent: Competency Constructor Help
 - Degree
 - Documentation
 - Employability
-- Notes
\ No newline at end of file
+- Notes
+=======
+
+>>>>>>> 4ee7da073bd147ad797fe3e9bf64d6a2598bc09c
```
Comment (mfranz1) [inline editCompetency.md:21]:
```suggestion

```

File: editCompetency.md (hunk 1 above)
Comment (mfranz1) [inline editCompetency.md:1]:
```suggestion
```

File: resetPassword.md (hunk 2 above)
Comment (mfranz1) [inline resetPassword.md:1]:
```suggestion
```

**Review by JosueMolinaMorales — CHANGES_REQUESTED** [review]
The previous changes havent been addressed and you will need to install Jekyll to properly setup the repo. Here is the homepage to Jekyll https://jekyllrb.com/
you should look into the installation guide and creating the project. You should see a _config.yml file after and the structure should resemble that of cyber4all docs and competency docs. let me know if you have any questions.

**Review by Cwagne17 — COMMENTED** [review]
Once Josue's and Mike's comments are addressed rerequest.

**Review by pzalep1 — COMMENTED** [review]
Once Josue's and Mike's comments are addressed you will be good!

**Pull request closed without merging**